CHANGELOG.md

```diff
@@ -10,6 +10,61 @@ Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfl
 -->
 
 
+## [0.10.0] - 2022-02-11
+
+## Highlights
+
+Major changes include:
+
+* Dropping support of DBT < 0.2.0 ** BREAKING CHANGE **
+* `sqlfluff fix` no will no longer fix SQL containing parsing or templating errors ** BREAKING CHANGE **
```

Comment:
Should we mention the new `--fix-even-unparsable` option here, in case people don't like the change?  I'm a little bit unsure about this, because, while some people may want it, advertising it may cause people to use it even though we'd rather they didn't. 🤔

Reply:
I say now. Those that really want it can find it in the docs, or raise an issue. Let's not advertise it.